### PR TITLE
Feat/16445: Remove network prefixes from Wallet Settings and Profile

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

This PR implements the new requirements to remove address prefixes across the app
https://www.notion.so/Remove-all-address-prefixes-eth-opt-arb-1128f96fb65c802780e2c10fa0b561fe

The parts mentioned in the UserStory are covered. Except:
1. SendModal adding an address with prefixes. This will be removed when we work on new send modal simplifications. Not removed here because of dependencies with advanced/custom view which will anyways be removed during the simplification.
2. Same as above for the Bridge Modal

### Affected areas

1. Saved Addresses
2. Wallet Settings
3. Profile showcase
4. Send Modal
5. Bridge Modal
6. Swap Modal
7. Receive Modal

### Architecture compliance

- [x] I am familiar with the application architecture and agreed good practices.
My PR is consistent with this document: [Status Desktop Architecture Guide](https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md)

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

### CODE REVIEW

**Review by alexjba — APPROVED** [review]
Beautiful! 😄 

Please consider updating the squish tests and status-go if needed.

**Review by alaibe — APPROVED** [review]

**Comment by anastasiyaig** [comment]
i am checking finally on this PR @Khushboo-dev-cpp , sorry for delay

File: ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml [hunk 1]
```diff
@@ -68,9 +64,7 @@ StatusModal {
         if (d.addressInputIsENS)
             addressInput.setPlainText(d.ens)
         else
-            addressInput.setPlainText("%1%2"
-                                      .arg(d.chainShortNames)
-                                      .arg(d.address == Constants.zeroAddress? "" : d.address))
+            addressInput.setPlainText("%1".arg(d.address == Constants.zeroAddress? "" : d.address))
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml:67]:
```suggestion
            addressInput.setPlainText(d.address == Constants.zeroAddress? "" : d.address)
```

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**